app/controllers/dashboards_controller.rb

```diff
@@ -22,9 +31,29 @@ def export
     redirect_to dashboards_path, alert: e.message
   end
 
+  def push
+    version = Version.preview_version
+
+    if version.blank?
+      flash.alert = 'No preview version available'
+    else
+      pv = Version.create(number: version.number, production: true, user: current_user)
+
+      if pv.persisted?
+        flash.notice = 'Production data updated'
+      else
+
+        flash.alert = "Production data not updated: #{version.inspect} #{pv.errors.inspect}"
+      end
+    end
+
+    redirect_to dashboards_path
+  end
+
   private
 
   def csv_model(csv_type)
+    return Institution if csv_type == 'Institution'
```

Comment:
I think that is covered by this spec:
```
    it 'redirects to index on error' do
      expect(get(:export, csv_type: 'BlahBlah', format: :csv)).to redirect_to(action: :index)
      expect(get(:export, csv_type: 'Weam', format: :xml)).to redirect_to(action: :index)
    end
```
Anything that is not in the InstitutionBuilder::TABLES or 'Institution' should return nil, and that will trip a`raise StandardError, 'No data to export.'`. I could specialize the error message to say something along the lines of Non-supported CSV...